src/store/states/nav/nav-state.ts

```diff
@@ -0,0 +1,11 @@
+export interface INestedNavState {
+  title?: string;
+}
+
+// TODO: This needs to be localized
+export enum NavTabName {
```

Comment:
Could we import this from where we declare the tab names instead of declare the strings again? Just so that we dont have the same string in different places

Reply:
will look into